sdk/core/azure-core/src/main/java/com/azure/core/util/logging/LoggingUtils.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.logging;
+
+import com.azure.core.util.CoreUtils;
+
+class LoggingUtils {
+    private static final String NEW_LINE = System.lineSeparator();
```

Comment:
This won't always be `\r\n` depending on the OS being used and may break if `\r\n` is used in an OS that uses `\n`, or more likely if `\n` is used on a `\r\n` OS.